marlowe-tutorial/doc/marlowe-plutus.adoc

```diff
@@ -219,7 +219,5 @@ ____
 
 === Where to go to find out more
 
-* The PlutusTX tutorial
-https://prod.playground.plutus.iohkdev.io/tutorial/#plutus-tx[link]
-* The Wallet API tutorial
-https://prod.playground.plutus.iohkdev.io/tutorial/#validator-scripts[link]
+* https://prod.playground.plutus.iohkdev.io/tutorial/#plutus-tx[The PlutusTX tutorial]
+* https://prod.playground.plutus.iohkdev.io/tutorial/#validator-scripts[The Wallet API tutorial]
```

Comment:
These links are also a bit sketchy... it would be nice to feed this thing a link to the plutus playground. We can do the same thing that we do for the marlowe playground link, namely:
- have an attribute for it with some sensible default value.
- override it on the command line.
- use suitable values to glue it all together for real deployments.

Reply:
pushed something and tested on https://david.marlowe.iohkdev.io/tutorial/

this page isn't actually published but there is a link at the bottom of section 1 which directs to the marlowe playground using the variables.